src/main/java/org/eclipse/yasson/internal/JsonbContext.java

```diff
@@ -130,4 +141,23 @@ public JsonbConfigProperties getConfigProperties() {
     public InstanceCreator getInstanceCreator() {
         return instanceCreator;
     }
+
+    private JsonbComponentInstanceCreator initComponentInstanceCreator(InstanceCreator instanceCreator) {
+        ServiceLoader<JsonbComponentInstanceCreator> loader = AccessController
+                .doPrivileged((PrivilegedAction<ServiceLoader<JsonbComponentInstanceCreator>>) () -> ServiceLoader
+                        .load(JsonbComponentInstanceCreator.class));
+        List<JsonbComponentInstanceCreator> creators = new ArrayList<>();
+        for (JsonbComponentInstanceCreator creator : loader) {
+            creators.add(creator);
+        }
+        if (creators.isEmpty()) {
+            // No service provider found - use the defaults
+            return JsonbComponentInstanceCreatorFactory.getComponentInstanceCreator(instanceCreator);
+        }
+        creators.sort(Comparator.comparingInt(JsonbComponentInstanceCreator::getPriority).reversed());
+        JsonbComponentInstanceCreator creator = creators.get(0);
+        log.finest(Messages.getMessage(MessageKeys.SERVICE_PROVIDER_COMPONENT_INSTANCE_CREATOR, creator.getClass()));
```

Comment:
using an external message resource isn't necessary for a log.finest statement, just INFO or higher messages and exception messages need that.

Reply:
Ok. I was just following the usage from [JsonbComponentInstanceCreatorFactory](https://github.com/eclipse-ee4j/yasson/blob/master/src/main/java/org/eclipse/yasson/internal/components/JsonbComponentInstanceCreatorFactory.java#L58).